local-tests/test_catch_up.py

```diff
@@ -46,10 +58,10 @@ def printt(s): print(ctime() + ' | ' + s)
 
 chain.start('aleph', nodes=[4, 5])
 
-printt('Waiting for finalization')
-chain.wait_for_finalization(0)
 printt('Waiting for authorities')
 chain.wait_for_authorities()
+printt('Waiting for finalization')
+chain.wait_for_finalization(90)
```

Comment:
Why 90?

Reply:
Because we want pruning to start. Changed that to be more clear